include/extractor/raster_source.hpp

```diff
@@ -4,16 +4,23 @@
 #include "util/coordinate.hpp"
 #include "util/exception.hpp"
 
+#include <boost/algorithm/string.hpp>
 #include <boost/algorithm/string/trim.hpp>
 #include <boost/assert.hpp>
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/fstream.hpp>
+#include <boost/foreach.hpp>
 #include <boost/spirit/include/qi.hpp>
 #include <boost/spirit/include/qi_int.hpp>
+
 #include <storage/io.hpp>
 
+#include <iostream>
 #include <iterator>
+#include <list>
```

Comment:
seems the header is not used. Is this a left-over from debugging?

Reply:
Thanks for your check and comment.
I will remove it after confirmation.